Extension/src/LanguageServer/client.ts

```diff
@@ -952,6 +1014,7 @@ export class DefaultClient implements Client {
                         vscode.window.showErrorMessage(localize("unable.to.start", "Unable to start the C/C++ language server. IntelliSense features will be disabled. Error: {0}", String(err)));
                     }
                 }));
+            this.colorizationSettings.reload();
```

Comment:
>this.colorizationSettings.reload(); [](start = 12, length = 35)

is this a bug fix?

Reply:
I move this out of the constructor, so that it could be constructed earlier and stored without using `| undefined`.